.github/workflows/build-and-test.yaml

```diff
@@ -111,20 +110,17 @@ jobs:
           elixir_version: "1.7"
           cmake_opts: "-DOPENSSL_CRYPTO_LIBRARY=/usr/lib/i386-linux-gnu/libcrypto.so"
           arch: "i386"
-          compiler_pkgs: "gcc-4.8 g++-4.8 gcc-4.8-multilib g++-4.8-multilib libc6-dev-i386
-          libc6-dbg:i386 zlib1g-dev:i386 libssl-dev:i386"
+          compiler_pkgs: "gcc-4.8 g++-4.8 gcc-4.8-multilib g++-4.8-multilib libc6-dev-i386 libc6-dbg:i386 zlib1g-dev:i386 libssl-dev:i386"
 
         - os: "ubuntu-20.04"
           cc: "gcc-10"
           cxx: "g++-10"
           cflags: "-m32 -O3"
           otp: "23.0"
           elixir_version: "1.9"
-          cmake_opts: "-DOPENSSL_CRYPTO_LIBRARY=/usr/lib/i386-linux-gnu/libcrypto.so
-          -DAVM_DISABLE_FP=on"
+          cmake_opts: "-DOPENSSL_CRYPTO_LIBRARY=/usr/lib/i386-linux-gnu/libcrypto.so -DAVM_DISABLE_FP=on"
           arch: "i386"
-          compiler_pkgs: "gcc-10 g++-10 gcc-10-multilib g++-10-multilib libc6-dev-i386
-          libc6-dbg:i386 zlib1g-dev:i386 libssl-dev:i386"
+          compiler_pkgs: "gcc-10 g++-10 gcc-10-multilib g++-10-multilib libc6-dev-i386 libc6-dbg:i386 zlib1g-dev:i386 libssl-dev:i386"
```

Comment:
I usually try to keep stuff under 100 columns as much as possible.
I realize that most of the changes here above are just reformatting noise, if you don't mind I would revert the part here above.